message-channel/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: pattern
+title: Message Channel
+folder: message-channel
+permalink: /patterns/message-channel/
+categories: Integration
+tags: Java
+---
+
+**Intent:** When two applications communicate using a messaging system they do it by using logical addresses
+of the system, so called Message Channels.
+
+![alt text](./etc/message-channel.png "Message Channel")
+
+**Applicability:** Use the Message Channel pattern when
```

Comment:
I wouldn't leave it out. We should strive to have content for the fields mentioned in the [pattern template](https://github.com/iluwatar/java-design-patterns/wiki/2.-Pattern-template).